blinkt.go

```diff
@@ -93,17 +119,39 @@ func (bl *Blinkt) Show() {
 }
 
 // SetAll sets all pixels to specified r, g, b colour. Show must be called to update the LEDs.
-func (bl *Blinkt) SetAll(r int, g int, b int) {
+func (bl *Blinkt) SetAll(r int, g int, b int, brightness ...float64) {
```

Comment:
Method chaining looks interesting.  I will investigate further.  I think we'd need a SetPixelBrightness method too.

`blinkt.SetPixel(1,255,0,0).SetPixelBrightness(1, 0.1)`